docs/beginning-pyscript.md

```diff
@@ -106,7 +106,7 @@ module in the document's `<head>` tag:
       <meta charset="utf-8" />
       <meta name="viewport" content="width=device-width,initial-scale=1" />
       <title>🦜 Polyglot - Piratical PyScript</title>
-      <script type="module" src="https://pyscript.net/snapshots/2023.11.1.RC3/core.js"></script>
+      <script type="module" src="https://pyscript.net/snapshots/2023.11.1/core.js"></script>
```

Comment:
random comment, but ideally we should have a script which automatically updates all these URLs.
It's only a matter of time before we will forget to update one

Reply:
Totally agree... it's on my todo list.